protocol-units/execution/opt-executor/src/executor/transaction_pipe.rs

```diff
@@ -45,10 +44,17 @@ impl Executor {
 			match request {
 				MempoolClientRequest::SubmitTransaction(transaction, callback) => {
 					// For now, we are going to consider a transaction in flight until it exits the mempool and is sent to the DA as is indicated by WriteBatch.
-					let in_flight = self.transactions_in_flight.load(std::sync::atomic::Ordering::Relaxed);
-					info!("Transactions in flight: {:?}", in_flight);
-					if in_flight > 2^16 {
-						info!("Shedding load.");
+					let in_flight = self.transactions_in_flight.load(std::sync::atomic::Ordering::SeqCst);
+					info!(
+						target: "movement_timing",
+						in_flight = %in_flight,
+						"transactions_in_flight"
+					);
+					if in_flight > 2u64.pow(16) {
+						info!(
+							target: "movement_timing",
+							"shedding_load"
+						);
```

Comment:
I don't know if we should be using the timing target for individual events that much. Is this data helpful in the timing log?